forge-gui/res/cardsfolder/upcoming/solitary_study_endless_corridor.txt

```diff
@@ -0,0 +1,16 @@
+Name:Solitary Study
+ManaCost:1 W
+Types:Enchantment Room
+AlternateMode:Split
+S:Mode$ Continuous | Affected$ Creature.YouCtrl | AddPower$ 1 | Description$ Creatures you control get +1/+0.
+Oracle:Creatures you control get +1/+0.
+
+ALTERNATE
+
+Name:Endless Corridor
+ManaCost:1 W
+Types:Enchantment Room
+T:Mode$ UnlockDoor | ValidPlayer$ You | ValidCard$ Card.Self | Execute$ TrigConjure | TriggerDescription$ When you unlock this door, conjure a duplicate of this card into your hand. When you do, target creature you control gains first strike until end of turn.
+SVar:TrigConjure:DB$ MakeCard | Conjure$ True | DefinedName$ TriggeredCard | Zone$ Hand | SubAbility$ DBPump
+SVar:DBPump:DB$ Pump | ValidTgts$ Creature.YouCtrl | KW$ First Strike | TgtPrompt$ Select target creature you control | SpellDescription$ Target creature you control gains first strike until end of turn.

```

Comment:
"when you do" is reflexive
use ImmediateTrigger

Reply:
Oh wait I did not mean to commit this to this branch! I am so sorry.